tests/test-javalin-jsonb/src/test/java/io/avaje/http/generator/JavalinProcessorTest.java

```diff
@@ -177,6 +177,41 @@ public void testOpenAPIGeneration() throws Exception {
     assertThat(generatedOpenApi).isEqualTo(expectedOpenApiJson);
   }
 
+  @Test
+  public void testOpenAPIGenerationMarkdown() throws Exception {
+    final var source = Paths.get("src").toAbsolutePath().toString();
+    // OpenAPIController
+    final var files = getSourceFiles(source);
+
+    Iterable<JavaFileObject> openAPIController = null;
+    for (final var file : files) {
+      if (file.isNameCompatible("OpenAPIControllerMarkdown", Kind.SOURCE))
+        openAPIController = List.of(file);
+    }
+    final var compiler = ToolProvider.getSystemJavaCompiler();
+
+    final var task =
+        compiler.getTask(
+            new PrintWriter(System.out),
+            null,
+            null,
+            List.of("--release=11", "-AdisableDirectWrites=true"),
```

Comment:
release 23 minimum if you want markdown

Reply:
you can go ahead and change that - if you can get this test passing, I think it's good to have?
Up to you whether you think it's good to have a direct test for markdown

also, there is a direct copy of this in tests for sigma generation etc too

Reply:
I mean the method we use to get the javadoc is the same for both non markdown and markdown so I don't really see the point